rundeckapp/src/test/groovy/org/rundeck/app/components/JobYAMLFormatSpec.groovy

```diff
@@ -264,6 +264,93 @@ class JobYAMLFormatSpec extends Specification {
         result[0].notification['onsuccess'].findAll{ it['urls'] != null }.size() == 2
     }
 
+    @Unroll
+    def "should return a notification map webhook url notifs"() {
```

Comment:
The decode method always returns list of notifications for each trigger. Example of better name: `"should return a notif list with one webhook notif"`

Reply:
Test - renamed